api/specs.proto

```diff
@@ -264,6 +264,9 @@ message ContainerSpec {
 	// task will exit and a new task will be rescheduled elsewhere. A container
 	// is considered unhealthy after `Retries` number of consecutive failures.
 	HealthConfig healthcheck = 16;
+
+	// SecurityProfiles specifies per-architecture security options required by the container
+	SecurityProfiles security_profiles = 21;
```

Comment:
Is this the right wording? My understanding was that security profiles would be high-level privilege grants, but `ServiceSpecs` would contain low level capabilities.